content/post/36-a-look-at-windows.md

```diff
@@ -0,0 +1,52 @@
+---
+date: 1970-01-01
+url: 36-a-look-at-windows
+title: A look at go-ipfs on Windows
+author: Dominic Della Valle
+---
+
+`go-ipfs` is built on top of [Golang](https://golang.org/), which allows for some cross platform compatibility, however it doesn't cover everything. Let's take a look at some of the issues of running `go-ipfs` on Windows and the upcoming solutions for them.
+
+### Log output
+![colour](img/colour.png)  
+Previously, the output on Windows was filled with non-native [control characters](https://en.wikipedia.org/wiki/Control_character) which made it hard to read, not only for users but also for developers when users would share their logs for debugging.  
+
+Now, there should be no more oddities related to character color or cursor placement, text should be clear and lines shouldn't overlap anymore. Which should make everyone a little bit happier.
+
+### Building
+![build](img/build.gif)  
+Previously, there were numerous issues with building the Windows binary, on Windows itself. Silent failures, lack of respect for user supplied arguments, inconsistent handling of dependencies, and others. For a more in-depth look at them, feel free to check out the [Windows initiative issue](https://github.com/ipfs/go-ipfs/issues/4808) which references them.  
+
+Now, the building experience should be consistent with other platforms, in addition the [documentation has been rewritten](https://github.com/ipfs/go-ipfs/blob/master/docs/windows.md), clarifying the process and adding a section that covers Windows specific concerns and how to deal with them.
+
+### Access errors / Garbage files {TODO: more appropriate word than "garbage"}
+![garbage](img/garbage.png)  
+
+Previously, there was a particularly bad bug that often caused failure for many flatfs (`go-ipfs`'s default storage method) operations, sometimes garbage files would be produced in the working directory as well.  
+The details of which can be found in the [Windows initiative issue](https://github.com/ipfs/go-ipfs/issues/4808), the short version is that we were trying to move data blocks from a temporary location to a target location, but the target location was being mangled into garbage. {TODO: still sounds too technical; see if we can avoid the word "mangled" as well}
```

Comment:
I think you could simplify this by just stating the issue first, then linking somewhere for more details second:

> When trying to move data blocks from a temporary location, the destination path was often mangled. For deeper details, see the Windows initiative issue.